test/e2e/gitops_service_test.go

```diff
@@ -237,3 +244,97 @@ func tearDownArgoCD(t *testing.T) {
 	assertNoError(t, err)
 
 }
+
+func policyRulesForClusterConfig() []rbacv1.PolicyRule {
+	return []rbacv1.PolicyRule{
```

Comment:
For every resource type that's listed here, we need to have one test that creates an Argo `Application` CR and consumes manifests.